### ISSUE TITLE
revision to 'sustainable development target'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#59]
Propose that `sustainable development target` be moved under IAO `objective specification`
and defined as

An objective specification which describes the endpoint of a sustainable development process and is part of a sustainable development goal.

**Comment by pbuttigieg** [comment]
The current def is:
>A threshold value of a sustainable development indicator or a set of threshold values of several such indicators towards or beyond which corresponding indicator values from a socio-economic-environmental system should tend in order to meet a sustainable development goal.

The thinking here is that indicators (see #58) are actual parts of a system which can be examined to inform us about that system's health. This is done by performing some OBI planned processes to observe and create data (indicator values SDGIO_00000003) about those indicators. A target is a value towards which those indicator values should tend. If they do, it should mean that the objective specification of some SDG is being (partially) met.

Targets, therefore, are semantically close to indicator values in that we assume that they are the indicator values of a system that has achieved some SDG. I'm not sure how to treat a data item which is about a hypothetical system yet.

@phismith proposed a framework for targets and indicators, including a dimension along which indicator values and targets could be distributed.

**Comment by cfrancois7** [comment]
Why not defined the sustainable development target as a process.
The purpose is to achieve a sustainable development process (process) that meets a sustainable development goal (objective specification). The _sustainable development target_s could be the _process profile_s of the _sustainable development process_.

I know this proposition is far from the current definition.

**Comment by pbuttigieg** [comment]
@cfrancois7 Thanks for your input!

We currently see targets as being more on the data/information side as most of them reference some threshold (e.g. X% employment) which is believed to coincide with a value that would indicate a desirable system is in place.  I don't see how this could be a process (however, the processes which drive the system towards/away from that target must also be represented). 

Could you give an example of how your thinking would play out using one or more of the SDG targets?

**Comment by phismith** [comment]
On Fri, Oct 28, 2016 at 8:53 AM, cfrancois7 <notifications@github.com>
wrote:

> Why not defined the sustainable development target as a process.
> The purpose is to achieve a sustainable development process (process) that
> meets a sustainable development goal (objective specification). The *sustainable
> development target_s could be the _process profile_s of the _sustainable
> development process*.
>
​This just pushes the issue back one step (and adds an extra and unhelpful
layer of complexity)

We want to measure process towards a target
The target is creating another process (call it a cfrancois7-process)
But then that 2nd process has to have a target
(I suppose that, too, might be a process (perhaps a cfrancois7-squared
process)​

**Comment by cfrancois7** [comment]
Yes, I agree it adds an extra layer. In the approach I proposed, the targets are the measures of the process profiles of a process that is a sustainable process, e.g. this process will meet one or more sustainable development goals. For instance, It comes from an experimentation or a simulation and the instance is called the "reference sustainable process".
Now, if the measures of the process profiles of a second process are higher (or lower) of this reference sustainable process, then it is a sustainable process by inferences.

My example is inspired from the acid/basic, redox or thermodynamic chemical reactions. Through a lot of experiments or models, it is now possible to draw the diagrams (several process profiles of several experimental processes) on which we define the thresholds of interests, e.g. the ratio of oxidation reactions on reduction ones that induces a dissolution/precipitation of the salt in water, same thing with gas/liquid/solid thresholds for water under one atmosphere.

So, the approach I proposed is more to create a "targeted process profile of a sustainable process" than "sustainable development target". The information is store in the process profiles themselves.

The issue in this approach is the need to define one or several "sustainable process" to define the targets and assess the other processes to these ones.

**Comment by phismith** [comment]
I think the problem with this is that 'Targets' are meant to be easily
measurable
BS

**Comment by cfrancois7** [comment]
Before to talk about the threshold value, I prefer to define the threshold. I used documents about the sustainable development but ecological oriented.

Quotes from the [Seri' study for the European Union,](http://ec.europa.eu/environment/enveco/waste/pdf/thresholds_final_report.pdf)

> The  thresholds  at  stake  can  be  directly  linked  to  a  state  of  the  environment  (e.g.  biodiversity  level),  a  pressure  (e.g.  emissions)  and/or  to  an  underlying driving force (e.g. social or economical) made on this state or pressure.

> A tolerable level of soil erosion refers to the amount of soil that can be lost per area and year without undermining  the  soil’s  capacity  to  regenerate.  This  would  constitute  a  potential  threshold  for  maintaining soil quality

When I read the document, it was clear the threshold is considered as an event, a limit in the process, sometimes called boundary. Thus, one or several indicators were created to assess the phenomenon or give a threshold indicator value to help the policy makers. But the thresholds were always define through models or empirical studies. The approach is: `review of models/ scientific literature  => threshold phenomena => threshold definition => indicator definition to translate the threshold in few figures and easy measurable => threshold indicator value`.

So, if you have one `threshold indicator value (the target)`, this value is based on one threshold phenomena in a `process`, which is an event and a kind `boundary process`. Thus, an `indicator` is created to translate the `target` that represents the natural irreversible limit.

About the indicator, I see it as `an objective specification` on the based of several examples in the document (above), e.g. _observed maximum concentration of nitrogen at monitoring site_ or _Ratio of observed to maximum allowable concentration_  or _non-renewable water use_. Also, the indicator doesn't always give directly a state of the system but it is clearly defined to translate the threshold in few figures and in a easy way, e.g. non-renewable water consumption.

I proposed that a `sustainable development target` is about the `sustainable development threshold` that `is a` `boundary process` and `has part` one or a data set of measurement datum. The indicator could be defined as an `objective specification` because the action specification depends on each study case and the indicator doesn't inform on the real state (e.g. a quality) of the system. The indicator is create to translate the threshold into value.

What is your point of view? Supposedly I read correctly the document.
